build.gradle

```diff
@@ -28,15 +28,18 @@ allprojects {
     }
 
     project.ext {
-        awsSdkVersion = '1.11.871'
-        awsSdk2Version = '2.14.16'
-        nvaDatamodelVersion= "1c1c6aaec0c56b25b1eeca6ee08dedf873cfa801"
+        awsSdkVersion = '1.11.972'
+        awsSdk2Version = '2.16.16'
+        nvaDatamodelVersion = '1c1c6aaec0c56b25b1eeca6ee08dedf873cfa801'
+        nvaDatamodelVersion= "0.11.6"
```

Comment:
are you mixing a hash version and a regular version in the same variable nvaDatamodelVersion?

Reply:
oops